<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WVectorSelector</class>
 <widget class="QWidget" name="WVectorSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>255</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WVectorSelector</string>
  </property>
  <widget class="QGroupBox" name="g_vector_comp">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>280</width>
     <height>91</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>280</width>
     <height>90</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>280</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="title">
    <string>Vector components</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="in_component">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>21</height>
         </size>
        </property>
        <property name="text">
         <string>Component</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="c_component">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>21</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <item>
         <property name="text">
          <string>radial</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>tangential</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>axial</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>comp_x</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>comp_y</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>comp_z</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="in_referential">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>21</height>
         </size>
        </property>
        <property name="text">
         <string>Referential</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="c_referential">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>21</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>xyz</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>radphiz</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
